app/javascript/src/components/TutorialPage.tsx

```diff
@@ -106,11 +106,19 @@ const pages: TutorialPageContent[] = [
     link: 'Get additional help.'
   },
   {
-    title: '#11 - Time to Play!',
+    title: "#11 - Advanced - Copying and Pasting",
+    text: [
+      'If you want certain sections of your story to also be in another one of your stories, you can easily copy and paste that section of the story with either our \'Copy\' and \'Paste\' buttons or your typical keyboard shortcuts.',
+      'In order to select the section of your story to copy, you have several options. You can just click on a singular scene you want to copy, you can hold the Shift key and drag your mouse over the seciton you want to copy, or you can hold the Shift key while clicking the scenes you wish to copy.'
+    ],
+    image: require('../images/tutorial/copy-paste.gif')
+  },
+  {
+    title: '#12 - Time to Play!',
     text: ['Press “Play” to read your story and get a shareable link.']
   },
   {
-    title: '#12 - Disclaimer',
+    title: '#13 - Disclaimer',
     text: [
       "This project was primarily built in a weekend, so you may encounter some quirks along the way. If something doesn't look right, saving your story and refreshing the page might do the trick. If saving isn't working, you can export your story as a last resort.",
```

Comment:
There's a line here about exporting your story as a last resort, we should pull that.

And we _could_ consider getting rid of this disclaimer at this point. We're a few major rounds past "this was built in a weekend", although I guess the word "primarily" holds that sentence down. You have any thoughts?

Reply:
I would say the sentence is still technically true, so there wouldn't be anything completely wrong with leaving it, but you could phrase it as stretching the truth at this point and could possibly have a slight moral objection to it, but I could go either way honestly. If I am already going to be in there to remove the export line, it would not be difficult for me to change that one as well though.

Reply:
Up to you man. I have no strong opinion here.

Reply:
I decided to leave the weekend part there. I do not think it is a lie and it maybe makes people less likely to get super mad about a bug in feedback.

Reply:
👍  good thinking